docs/intro.rst

```diff
@@ -0,0 +1,103 @@
+            #. Store updated offsets in HDFS.
+    #. Cleanup stage:
+        Read counts from all tasks, aggregates the values,
+        and submits the results to Kafka for consumption by other systems.
+
+Quickstart 
+----------
+
+Assuming you have confluent platform installed.
+
+.. sourcecode:: bash
+
+   # Assuming that a Hadoop cluster is already deployed and you have access to it
+   # Start a testing local Kafka cluster (1 Zookeeper node, 1 Kafka node)
+   $ cd kafka
+   $ bin/zookeeper-server-start.sh config/zookeeper.properties
+   $ bin/kafka-server-start.sh config/server.properties
+   # Create testAvro topic in Kafka
+   $ bin/kafka-topics.sh --create --zookeeper localhost:2181 \
+         --topic testAvro --partitions 1 --replication-factor 1
+   $ cd ..
+
+   # Start Schema Registry. The default schema registry settings
+   # automatically work with the default settings of ZooKeeper and Kafka.
+   $ cd schema-registry
+   $ bin/schema-registry-start config/schema-registry.properties
+   $ cd ..
+
+   # Start Kafka rest proxy.
+   $ cd kafka-rest
+   $ bin/kafka-rest-start kafka-rest.properties
+   $ cd ..
+
+   # Assuming that you have hadoop on your PATH, and make sure that
+   # schema.registry.url points to the correct address
+   $ cd camus
+   $ hadoop jar confluent-camsu-0.1.0-SNAPSHOT-shaded.jar com.linkedin.camus.etl.kafka.CamusJob \
+     -P etc/camus.properties
+
+Installation
+------------
+
+Release versions are available from the `Central
+Repository <http://search.maven.org/#search|ga|1|g%3A%22io.confluent%22%20AND%20a%3A%22camus%22>`_.
+
+Deployment
+----------
+Camus can be run from the command line. You will need to set some properties either by specifying a
+properties file on the classpath using -p (filename),or an external properties file using -P
+(path to local file system, or to hdfs),
+or from the command line using -D property=value.
+If the same property is set with multiple methods,
+the order of precedence is command-line properties, external properties file and
+classpath properties file.
+
+.. sourcecode:: bash
+
+   $ hadoop jar confluent-camsu-0.1.0-SNAPSHOT-shaded.jar com.linkedin.camus.etl.kafka.CamusJob
+      -D <property=value>
+      -P <path to external properties file>
+      -p <path to properties file from classpath>
+
+Development
+-----------
+
+To build a development version, you may need a development versions of
+`io.confluent.common <https://github.com/confluentinc/common>`_, 
+`io.confluent.rest-utils <https://github.com/confluentinc/rest-utils>`_ and
+`io.confluent.schema-registry <https://github.com/confluentinc/schema-registry>`_.
+After installing ``common``, ``rest-utils`` and ``schema-registry`` into local Maven repository
+and compiling ``camus`` with Maven, you can run camus with Hadoop.
+
+Contribute
+----------
+
+- Source Code: https://github.com/confluentinc/camus
+- Issue Tracker: https://github.com/confluentinc/camus/issues
```

Comment:
We should put the upstream repository here. Unlike projects created by Confluent, this section should actually provide more details about where issues should go. We're always happy to accept them, but it's much better if they go to the right location in the first place. @nehanarkhede do you have thoughts on how we should best handle the interaction with upstream? Of course, the alternative is to take out this Contribute section, or just rename it something else -- again, this was generated from a template for open source READMEs, so in this doc it might not be as relevant.